_functions/synth_parameters/delay.md

```diff
@@ -2,4 +2,5 @@
 title: delay
 category: synth_parameters
 ---
-a pattern of numbers from 0 to 1. Sets the level of the delay signal.
+a pattern of numbers that set the initial level of the delay signal.  I.e. a value of one means the first echo will be as
```

Comment:
Hm, unfortunately it is a bit more complicated – I just had a look again. If you have the sc3plugins installed, the delay will be a `SwitchDelay`, which crossfades when you change the delaytime. If you haven't, `CombL` will be used, which doesn't crossfade. But `CombL` uses negative  decaytimes to specify a negative feedback parameter. Unfortunately, `SwitchDelay` doesn't do this (I think this should be changed, but well...).